docs/modules/ROOT/pages/how-tos/testing/integration/group-snapshots.adoc

```diff
@@ -0,0 +1,75 @@
+    test.appstudio.openshift.io/group-test-info: [{"namespace":"ws-sample-tenant","component":"component-sample-1","buildPipelineRun":"component-sample-1-on-pull-request-****","snapshot":"application-sample-****","pullRuestNumber":"1","repoUrl":"https://github.com/sample-org/sample-repo1"},{"namespace":"ws-sample-tenant","component":"component-sample-2","buildPipelineRun":"component-sample-2-on-pull-request-****","snapshot":"application-sample-****","pullRuestNumber":"1","repoUrl":"https://github.com/sample-org/sample-repo2"}] <4>
+spec:
+  application: application-sample
+  components: <5>
+    - name: component-sample-1
+      containerImage: quay.io/redhat-user-workloads/ws-sample-tenant/application-sample/component-sample1@sha256:0db0a473a6abf5c15c424ab07cfbd5c40c06622fe648d4fe6a6b6abc224a0d0c
+      source:
+        git:
+          url: https://github.com/sample-org/sample-repo1
+          revision: fa8b89274a61ef0f1c257b7a84c37aa2ec844109
+    - name: component-sample-2
+      containerImage: quay.io/redhat-user-workloads/ws-sample-tenant/application-sample/component-sample2@sha256:93bc2f55e2d36e50d9915fd89da7d0bfd2499f0d49becf7cd6ff3a25b40d68f6
+      source:
+        git:
+          url: https://github.com/sample-org/sample-repo2
+          revision: ff13da0159d2a556c1fc9859a0f19f6b4472c4ad
+----
+<1> The label `test.appstudio.openshift.io/type: group` that indicates this is a `group` snapshot.
+<2> The label `test.appstudio.openshift.io/pr-group-sha: bc3c81ad3d0140f3fe63c61a0b2389cb76663d226fbd28b00fb0d9f050fb21` that indicates the sha256 of pr group derived from branch name and is used to query the k8s resource build pipelinerun and snapshots belonging to the same pr group. Build pipelineruns and snapshots built for the branch with the same branch name have the same pr group.
+<3> The annotation `test.appstudio.openshift.io/pr-group: feature1` that indicates the readable pr group derived from branch name.
+<4> The annotation `test.appstudio.openshift.io/group-test-info` indicates the component snapshots info forming the group snapshot.
+<5> The list of components and their container images. All components updated by pull/merge request with the same pr group will be included. For components which are not updated, the data from Global Candidate List will be used.
+
+== Verification
+You can see integration test result running for `group` snapshot is posted back to git pull/merge request via checkRun/commitStatus/comment.
+
+image::groupsnapshot.png[role="border" alt="Test Report"]
+
+NOTE: For monorepo, since the code changes of multiple components are from the same pull/merge request, the integration test result will be posted back to git pull/merge request only once.
+
+== Troubleshoot
+Sometimes, you can't see integration test result is posted back to pull/merge request, it might be because the group snapshot is not created. You may check by the following steps to debug the reason/failure:
```

Comment:
Do users have a recourse if the group snapshot isn't created (i.e. can they manually create a group snapshot ... or just a normal snapshot)? Are you just indicating that it isn't created _yet_ or that something might have prevented it from getting created?

Reply:
They can't manually create group snapshot. I am just indicating that it isn't created yet or that something might have prevented it from getting created.